src/apify/_utils.py

```diff
@@ -308,67 +327,68 @@ class LRUCache(MutableMapping, Generic[T]):
 
     _max_length: int
 
-    def __init__(self, max_length: int) -> None:
+    def __init__(self: LRUCache, max_length: int) -> None:
         """Create a LRUCache with a specific max_length."""
         self._cache = OrderedDict()
         self._max_length = max_length
 
-    def __getitem__(self, key: str) -> T:
+    def __getitem__(self: LRUCache, key: str) -> Any:
```

Comment:
This seems wrong to me. Should return `T`

Reply:
Yeah, I can do that. However, then I'll either have to put there `# type: ignore` or remove `warn_return_any`. Because it says:
> Returning `Any` from function declared to return `"T"` Mypy [no-any-return](https://mypy.readthedocs.io/en/latest/_refs.html#code-no-any-return)

Reply:
how about a `cast`?

Reply:
Yeah, that sounds like a good way to go...